app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/SummaryProductFragment.java

```diff
@@ -141,6 +142,11 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         final State state = (State) intent.getExtras().getSerializable("state");
 
         final Product product = state.getProduct();
+
```

Comment:
can you add a check to display it if the user has at least one allergen in their Alert list ?

Reply:
Sure :).